src/regional_settings.cpp

```diff
@@ -574,6 +624,17 @@ void forest_biome_component::finalize()
     }
 }
 
+void forest_biome_terrain_dependent_furniture::finalize()
+{
+    for( const std::pair<std::string, int> &pr : unfinalized_furniture ) {
+        const furn_str_id fid( pr.first );
+        if( !fid.is_valid() ) {
+            continue;
```

Comment:
This case should announce itself loudly. It is most likely just a misspelling, the user will wonder why the JSON code does not work.

Reply:
Makes sense. There's actually quite a few of these in the regional settings that have the same problem, so I'll make a pass and get them all at the same time.